src/python/aqueduct_executor/operators/connectors/tabular/connector.py

```diff
@@ -1,11 +1,12 @@
 from abc import ABC, abstractmethod
-from typing import List
+from typing import Any, List
 
 import pandas as pd
 from aqueduct_executor.operators.connectors.tabular import extract, load
+from aqueduct_executor.operators.utils.enums import ArtifactType
 
 
-class TabularConnector(ABC):
+class StorageConnector(ABC):
```

Comment:
Is "storage connector" the terminology we're going with for all our non-github integrations? It's a little confusing for me as a developer since we already use the term "storage" a lot in the backend to refer to our result stores (eg. we write output contents to "storage" at output_content_path)

Reply:
That was my intention. `TabularConnector` no longer make sense because in case of S3 the file doesn't necessarily need to be tabular any more. Any better naming ideas?

Reply:
`DataConnector` actually seems a little more specific for me (since I would think code != data) and this step-sides the "storage" naming collision. Essentially "storage" can be something we own (eg. our S3, our filesystem) whereas "data" is always coming from the user's data integrations.